app/uk/gov/hmrc/softdrinksindustrylevy/models/EnumUtils.scala

```diff
@@ -22,27 +22,27 @@ import scala.language.implicitConversions
 /** Utility class for creating json formatters for enumerations.
   */
 object EnumUtils {
-  def enumReads[E <: Enumeration](`enum`: E): Reads[E#Value] = new Reads[E#Value] {
-    def reads(json: JsValue): JsResult[E#Value] = json match {
+  def enumReads[E <: Enumeration](`enum`: E): Reads[`enum`.Value] = new Reads[`enum`.Value] {
+    def reads(json: JsValue): JsResult[`enum`.Value] = json match {
       case JsString(s) =>
         try
```

Comment:
I know this code block hasn't been part of this PR. But, can we simplify this pls - convert it to conditional case handling.